apps/ex_shop/lib/ex_shop.ex

```diff
@@ -15,7 +15,8 @@ defmodule ExShop do
       # worker(ExShop.Worker, [arg1, arg2, arg3]),
 
       worker(Commerce.Billing.Worker, stripe_worker_configuration, id: :stripe),
-      worker(Commerce.Billing.Worker, braintree_worker_configuration, id: :braintree)
+      worker(Commerce.Billing.Worker, braintree_worker_configuration, id: :braintree),
+      worker(Commerce.Billing.Worker, new_payment_gateway_configuration, id: :new_payment_gateway)
```

Comment:
Id should be different to start same worker with different configuration.

Remember, id is optional parameter so can be skipped but workers won't start without it :)